### PR TITLE
add 111. Minimum Depth of Binary Tree

### PR DESCRIPTION
**Pull request by tkkawa**
Please check my code.

### CODE REVIEW

File: 111_minimum-depth-of-binary-tree.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        depth = 1
+        min_depth = float('inf')
+        stack = deque([[root, depth]])
+
+        while stack:
+            cur_root, depth = stack.pop()
+            if not cur_root.right and not cur_root.left:
+                min_depth = min(depth, min_depth)
+                continue
+            if cur_root.left:
```
Comment (ryuji0123) [inline 111_minimum-depth-of-binary-tree.py:33]: Stop this. Just check cur_root

Comment (ryuji0123) [reply 111_minimum-depth-of-binary-tree.py:33]: I said this before

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by tkkawa** [comment]
I modified the code.

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
use either queue and recursive function

**Comment by tkkawa** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request closed without merging**